model/group.cc

```diff
@@ -270,6 +270,14 @@ namespace minsky
           else
             intOp->intVar->controller.reset();
         }
+        
+    // need to deal with godley to set initial width and height. for ticket 1205
+    if (auto godley=dynamic_cast<GodleyIcon*>(it.get()))        
```

Comment:
Why is this set here in addItem? This will mean godleyIcons are resized when being transferred from one group to another.

Reply:
For some reason, the unit tests were failing before I did this. But it seems the those test failures had more to do with the changes I made in the Group class. So, I've removed it.

Reply:
I didn't know where to initialize the size of the Godley icon now that iconWidth and IconHeight are gone. They were initialized to be 100x100. So, I'm trying the approach you used in Item::resize():

	 // Set initial iWidth() and iHeight() to initial Pango determined values. This resize method is not very reliable. Probably a Pango issue. 

    float w=iWidth(width()), h=iHeight(height()), invZ=1/zoomFactor();   

in GodleyIcon:resize():

	float z=zoomFactor(), **iw=this->iWidth(this->width())**, **ih=this->iHeight(this->height())**, is=iconScale();

It seems to work. Committing now.